__tests__/web/keyboard/index.spec.ts

```diff
@@ -0,0 +1,495 @@
+import { Page } from 'puppeteer/lib/types';
+import { IsolatedWorld } from '../isolatedWorld';
+
+describe('keyboard', () => {
```

Comment:
cool. and we also need copy-and-paste case ?

Reply:
yes.